var/spack/repos/builtin/packages/util-linux/package.py

```diff
@@ -21,9 +21,17 @@ class UtilLinux(AutotoolsPackage):
     depends_on('python@2.7:')
     depends_on('pkgconfig')
 
+    # make it possible to use the libuuid package instead...
```

Comment:
Comment could be confusing to others or to our future selves.   Instead, how about:

```
# you may need to disable util-linux's libuuid if you need to link against the libuuid's libuuid
```